packages/nextjs/app/page.tsx

```diff
@@ -32,9 +33,12 @@ const Home: NextPage = () => {
         <div className="px-5">
           <h1 className="text-center mb-8">
             <span className="block text-2xl mb-2">Welcome to</span>
-            <span className="block text-4xl font-bold">Batch 2</span>
+            <span className="block text-4xl font-bold">Batch2</span>
           </h1>
-          <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
+          <Image height={250} width={250} alt="BG-Batch2 logo" className="cursor-pointer mx-auto" src="/logo.svg" />
```

Comment:
We could also add `"dark:invert"` to `className`  so that the image is more visible in dark mode  too without needing to add a separate image 
: 

![Screenshot 2024-02-18 at 9 58 01 PM](https://github.com/BuidlGuidl/batch2.buidlguidl.com/assets/80153681/a5695c1d-774f-4a6a-9058-d7b9be2fa78d)

We also will need to add 
`darkMode: ["class", "[data-theme='scaffoldEthDark']"],` below  this line to [allow tailwind](https://tailwindcss.com/docs/dark-mode) class handle dark mode too : 

https://github.com/BuidlGuidl/batch2.buidlguidl.com/blob/d19541d71bd4db8799bc38ecc62bd570168d71a6/packages/nextjs/tailwind.config.js#L5

If we like this we could also update it in the Header Image, what do you say @weslygio ? 

PS: I think we should add `darkMode: ["class", "[data-theme='scaffoldEthDark']"],` in `tailwind.conifg.js` by default in SE-2 cc @carletex ?  Maybe @weslygio can create a PR there too 🙌